sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TryStatementsWithIdenticalCatchShouldBeMerged.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
```

Comment:
I think it's worth starting the test cases with the compliant and non-compliant examples from the RSPEC, partly so it's clear whether the rule meets the minimal spec-ed behaviour, and also as a sanity check that the RSPEC examples are well-formatted and make sense. WDYT?

Reply:
I would like this check to be automated :) The unit tests are not much different than the RSPEC examples, or am I missing a test case?

Reply:
Yes, the RSPEC can change separately, but at least it's a point in time check that it was valid. And the code samples don't change very often.
The unit tests should definitely be far more extensive, but if the RSPEC examples are in the tests then it's a quick sanity that the rule is at least covering the obvious cases. Otherwise, you have to read the code samples in the RSPEC to check the expected behaviour, and then try to find the corresponding unit tests that cover the same logical case.